src/Invoker.ts

```diff
@@ -0,0 +1,114 @@
+import * as http from "http";
+import * as https from "https";
+import * as path from "path";
+import * as URL from "url";
+
+export class Invoker {
+    public static async invokeExpressFile(handler: string, port: number, jsonRequest: any, restOfPath?:string): Promise<any> {
+        const handlerParts = handler.split(".");
+        const fileName = handlerParts.join("/") + ".js";
```

Comment:
We should assume that "/" is the separator for files, not "." - and if we are just expecting a module name (as opposed to a module + function name), we do not need to add ".js" - node will load a module whether or not .js i added.

I think these two lines can basically just be taken out.